core/endpoint_test.go

```diff
@@ -398,6 +398,32 @@ func TestIntegrationEvaluateHealthWithError(t *testing.T) {
 	}
 }
 
+func TestIntegrationEvaluateHealthWithErrorAndHideURL(t *testing.T) {
+	condition := Condition("[STATUS] == 200")
+	endpoint := Endpoint{
+		Name:       "invalid-url",
+		URL:        "https://httpstat.us/200?sleep=60000",
+		Conditions: []*Condition{&condition},
+		UIConfig: &ui.Config{
+			HideURL: true,
+		},
+	}
```

Comment:
It would be nice to add a client config with a timeout of like `1ms`. By default, the timeout is 10s IIRC, and slow tests aren't very fun 😔

Reply:
Makes sense. 👌 Done!